dist/common/scripts/scylla_coredump_setup

```diff
@@ -85,8 +85,22 @@ WantedBy=multi-user.target
             run('sysctl -p /usr/lib/sysctl.d/50-coredump.conf', shell=True, check=True)
         else:
             with open('/etc/sysctl.d/99-scylla-coredump.conf', 'w') as f:
-                f.write('kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e"')
+                f.write(f'kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e')
             run('sysctl -p /etc/sysctl.d/99-scylla-coredump.conf', shell=True, check=True)
+        # Need to overwrite saved_core_pattern, otherwise stopping
+        # abrt-ccpp.service will modify kernel.core_pattern to 'core'
+        if os.path.exists('/var/run/abrt/saved_core_pattern'):
```

Comment:
instead rewriting the saved file with the existing settings to make `/usr/sbin/abrt-install-ccpp-hook uninstall` a no-op. probably, a simpler solution is to just remove `/var/run/abrt/saved_core_pattern`.

Reply:
Dropped previous code, rerwrite it to stopping/disabling abrt-ccpp.service.